Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -469,15 +466,15 @@ void CVolumeDirectory::WriteDirectory(const File::FSTEntry& parent_entry, u32* f
     {
       u32 entry_index = *fst_offset / ENTRY_SIZE;
       WriteEntryData(fst_offset, DIRECTORY_ENTRY, *name_offset,
-                     parent_entry_index << m_address_shift, entry_index + entry.size + 1);
+                     parent_entry_index, entry_index + entry.size + 1, 0ull);
```

Comment:
Why `0ull` instead of `0`? Is `unsigned long long int` really supposed to fit in `u32`? (Same for the `1ull` you're using further up in the file.)